code/obj/grille.dm

```diff
@@ -1,5 +1,5 @@
 /obj/grille
-	desc = "A sturdy metal mesh. Blocks large objects, but lets small items, gas, or energy beams through."
+	desc = "A metal mesh often built underneath windows to reinforce them. The holes can let small items, gas, and energy beams through."
```

Comment:
```suggestion
	desc = "A metal mesh often built underneath windows to reinforce them. The holes allow gas and energy beams to pass through it."
```
correct me if im wrong but no items can actually pass through a grille. keeping it more gameplay focused